site2/docs/functions-package-java.md

```diff
@@ -4,13 +4,33 @@ title: Package Java Functions
 sidebar_label: "Package Java Functions"
 ---
 
+There are two methods to package Java Functions, that is [uber JAR](#package-as-jar) and [NAR](#package-as-nar).
+
+:::note
```

Comment:
With the note and tip newly added, there will be three consecutive color blocks (but in different colors), which is a bit overwhelming in visuals IMO. Does it help by combining the three items in a single note/tip block?

Reply:
done